apps/community/backend/cmd/api/main.go

```diff
@@ -50,6 +49,7 @@ var NodeConfig = &ipfs.IPFSNodeConfig{
 */
 func CreateCommunity(name string, path string) (error, string, string, []string) {
 	err, key, peerid, addrs := NodeConfig.NewCommunityIPFSNode(name, path)
+	time.Sleep(5 * time.Second) // need to remove this at some point --> basically wait til ipfs comm is created before connecting
```

Comment:
Is it possible to make `NewCommunityIPFSNode` block until the community is done coming up? Might be hacky, I think the way we did it in the old habitat repo was just scanning the output until we saw "Daemon is ready" or something like that

Reply:
fixed!